.gitlab-ci.yml

```diff
@@ -1,21 +1,44 @@
-
 include:
-  - project: 'Northern.tech/Mender/mendertesting'
-    file: '.gitlab-ci-github-status-updates.yml'
+  - project: "Northern.tech/Mender/mendertesting"
+    file: ".gitlab-ci-github-status-updates.yml"
   - local: .gitlab-ci-staging-tests.yml
     rules:
       - if: $RUN_TESTS_STAGING == "true"
   - local: .gitlab-ci-default-pipeline.yml
     rules:
-      - if: $RUN_TESTS_STAGING != "true"
+      - if: $RUN_TESTS_STAGING == "true"
+        when: never
+      - if: $RUN_FULL_INTEGRATION == "true"
+        when: never
+  - local: .gitlab-ci-full-integration.yml
+    rules:
+      - if: $RUN_TESTS_FULL_INTEGRATION == "true"
 
 stages:
   - test
   - build
   - publish
 
 variables:
-  RUN_TESTS_STAGING: "false"
+  RUN_TESTS_STAGING:
+    description: "Run staging integration tests"
+    value: "false"
+  RUN_TESTS_FULL_INTEGRATION:
+    description: |
+      Run full integration tests.
+      NOTE: Can only run on protected branches
+    value: "false"
+    options:
+      - "false"
+      - "true"
+
+  DOCKER_VERSION:
+    description: "Docker version to use in jobs"
+    value: "27"
+
+  MENDER_TAG:
+    description: "Docker tag for running integration tests"
+    value: main
```

Comment:
What about the client? We should have a similar tag for it and update the docker-compose files accordingly

Reply:
True, but the task was about the `mender-server` so I tried to stick with it.

Reply:
Added `MENDER_CLIENT_TAG`.